From 97f0883e6001aa8d9a139944359707c391ad4ea7 Mon Sep 17 00:00:00 2001
From: "Erik M. Bray" <erik.bray@lri.fr>
Date: Fri, 19 Apr 2019 09:04:22 +0200
Subject: [PATCH] Trac #27702: Remove version parsing from Cygwin patch to
 psutil.

It currently is not used, and is broken on Cygwin 3.0.0+.
---
 build/pkgs/psutil/package-version.txt          |  2 +-
 build/pkgs/psutil/patches/cygwin-support.patch | 19 +------------------
 2 files changed, 2 insertions(+), 19 deletions(-)

diff --git a/build/pkgs/psutil/package-version.txt b/build/pkgs/psutil/package-version.txt
index 17b5de986f..3d61945d1e 100644
--- a/build/pkgs/psutil/package-version.txt
+++ b/build/pkgs/psutil/package-version.txt
@@ -1 +1 @@
-5.2.0.p1
+5.2.0.p2
diff --git a/build/pkgs/psutil/patches/cygwin-support.patch b/build/pkgs/psutil/patches/cygwin-support.patch
index 6ef41fb658..17eac80028 100644
--- a/build/pkgs/psutil/patches/cygwin-support.patch
+++ b/build/pkgs/psutil/patches/cygwin-support.patch
@@ -205,7 +205,7 @@ new file mode 100644
 index 0000000..ffba639
 --- /dev/null
 +++ b/psutil/_pscygwin.py
-@@ -0,0 +1,908 @@
+@@ -0,0 +1,891 @@
 +"""Cygwin platform implementation."""
 +
 +# TODO: Large chunks of this module are copy/pasted from the Linux module;
@@ -253,23 +253,6 @@ index 0000000..ffba639
 +# --- constants
 +# =====================================================================
 +
-+_cygwin_version_re = re.compile(r'(?P<major>\d+)\.(?P<minor>\d+)\.'
-+                                '(?P<micro>\d+)\((?P<api_major>\d+)\.'
-+                                '(?P<api_minor>\d+)/(?P<shared_data>\d+)/'
-+                                '(?P<mount_registry>\d+)\)')
-+_cygwin_version = _cygwin_version_re.match(os.uname()[2]).groupdict()
-+
-+CYGWIN_VERSION = (int(_cygwin_version['major']),
-+                  int(_cygwin_version['minor']),
-+                  int(_cygwin_version['micro']))
-+
-+CYGWIN_VERSION_API = (int(_cygwin_version['api_major']),
-+                      int(_cygwin_version['api_minor']))
-+
-+CYGWIN_VERSION_SHARED_DATA = int(_cygwin_version['shared_data'])
-+
-+CYGWIN_VERSION_MOUNT_REGISTRY = int(_cygwin_version['mount_registry'])
-+
 +
 +if enum is None:
 +    AF_LINK = -1
-- 
2.15.1

